README.md

```diff
@@ -217,6 +217,17 @@ disabled:
   Require all files to end with a newline character. One will be added if it is
   not present.
 
+* `:list-indent-size`
```

Comment:
This is a significant enough setting that it probably belongs higher up, directly under `:indentation?`.